.travis.yml

```diff
@@ -60,8 +61,9 @@ jobs:
       name: publish to npm
       deploy:
         provider: npm
+        edge: true
         email: $NPM_EMAIL
-        api_key: $NPM_API_KEY
+        token: $NPM_API_KEY
```

Comment:
The npm provider needs `api_token` - https://docs.travis-ci.com/user/deployment-v2/providers/npm/#known-options
You think they would be the same across all providers but apparently not!

Reply:
That would make too much sense, updated.